src/components/UserCollective.js

```diff
@@ -74,43 +74,45 @@ class UserCollective extends React.Component {
       },
       'organization.collective.edit': {
         id: 'organization.collective.edit',
-        defaultMessage: 'edit organization',
+        defaultMessage: 'edit Organization',
       },
       'user.collective.edit': {
         id: 'user.collective.edit',
         defaultMessage: 'edit profile',
       },
       'user.collective.memberOf.collective.host.title': {
         id: 'user.collective.memberOf.collective.host.title',
-        defaultMessage: "I'm hosting {n, plural, one {this collective} other {these {n} collectives}}",
+        defaultMessage: "I'm hosting {n, plural, one {this Collective} other {these {n} Collectives}}",
       },
       'user.collective.memberOf.organization.admin.title': {
         id: 'user.collective.memberOf.organization.admin.title',
-        defaultMessage: "I'm an administrator of {n, plural, one {this organization} other {these {n} organizations}}",
+        defaultMessage: "I'm an admin of {n, plural, one {this Organization} other {these {n} Organizations}}",
       },
       'user.collective.memberOf.organization.member.title': {
         id: 'user.collective.memberOf.organization.member.title',
-        defaultMessage: "I'm a member of {n, plural, one {this organization} other {these {n} organizations}}",
+        defaultMessage: "I'm a team member of {n, plural, one {this Organization} other {these {n} Organizations}}",
       },
       'user.collective.memberOf.collective.admin.title': {
         id: 'user.collective.memberOf.collective.admin.title',
-        defaultMessage: "I'm a core contributor of {n, plural, one {this collective} other {these {n} collectives}}",
+        defaultMessage:
+          "I'm a Core Contributor and admin of {n, plural, one {this Collective} other {these {n} Collectives}}",
       },
       'user.collective.memberOf.collective.member.title': {
         id: 'user.collective.memberOf.collective.member.title',
-        defaultMessage: "I'm a member of {n, plural, one {this collective} other {these {n} collectives}}",
```

Comment:
Is this really what you want? member -> Core Contributor

Reply:
My understanding of Member was that it referred to people in the section in 'edit Collective' that used to be called "Members" but has now been changed to "Core Contributors" whether they are admins or not. All the people who show in the 'team' section are now called 'Core Contributors' and there is no such thing as just 'Contributor' any more, and there's also no such thing as 'Member'.  Maybe I misunderstood what "member" is referring to in this instance?

Reply:
😅 

I'll check again.

Reply:
I confirm your understanding.